src/Symfony/Component/Debug/Exception/SilencedErrorContext.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Debug\Exception;
+
+/**
+ * Data Object that represents a Silenced Error.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class SilencedErrorContext implements \JsonSerializable
```

Comment:
I don't recall, why is it  `JsonSerializable`? (cc @nicolas-grekas)?

It does not play well with monolog: 
https://github.com/Seldaek/monolog/blob/b05bf55097060ec20f49ccec0cf2f8e5aaa468b3/src/Monolog/Formatter/NormalizerFormatter.php#L195-L197

I got this in my index:
```
{
  "_index": "monolog",
  "_type": "_doc",
  "_id": "sU9quogBWJfLYe0zo1VX",
  "_score": 1,
  "fields": {
    "context.exception.Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext.trace.type.keyword": [
      "->"
    ],
    "context.exception.Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext.trace.class.keyword": [
      "Symfony\\Component\\ErrorHandler\\ErrorHandler"
    ],
    "context.exception.Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext.trace.class": [
      "Symfony\\Component\\ErrorHandler\\ErrorHandler"
    ],
    "context.exception.Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext.trace.function": [
      "handleError"
    ],
    "context.exception.Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext.trace.function.keyword": [
      "handleError"
    ],
    "context.exception.Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext.trace.type": [
      "->"
    ],
    "context.exception.Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext.count": [
      1
    ],
    "context.exception.Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext.severity": [
      2
    ],
    "context.exception.Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext.file.keyword": [
      "Unknown"
    ],
    "context.exception.Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext.line": [
      0
    ],
    "context.exception.Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext.file": [
      "Unknown"
    ]
  }
}
```